src/MSBuild/CommandLineSwitches.cs

```diff
@@ -103,6 +103,7 @@ internal enum ParameterizedSwitch
             WarningsAsErrors,
             WarningsAsMessages,
             BinaryLogger,
+            Restore,
```

Comment:
Does Restore have any parameters? If it doesn't, shouldn't it go in the ParameterslessSwitch enum instead? Haven't read to code to understand the behavioural effects.

Reply:
It's technically `true|false` like `/nodeReuse`.  This allows you to have the environment variable set but still specify `/restore:false`.  `/restore` is mapped to `/restore:true`